src/components/ConfirmationModalWithOutcome.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
+import { Modal } from "react-bootstrap";
+import { Button } from "library-simplified-reusable-components";
+
+export const SHOW_CONFIRMATION = "requestConfirmation";
+export const SHOW_OUTCOME = "displayOutcome";
+export type ModalState =
+  | typeof SHOW_CONFIRMATION
+  | typeof SHOW_OUTCOME
+  | undefined;
```

Comment:
Because `SHOW_CONFIRMATION` and `SHOW_CONFIRMATION` are `const`s, their types resolve to literal types, so the `ModalState` type should resolve to
```
"requestConfirmation" | "displayOutcome" | undefined
```

It seemed like a nice way to avoid duplicating those as literals in the type, in case they needed to be changed.

Reply:
That makes sense, thanks.